### PR TITLE
UIkit Rewrite

### PR DESCRIPTION
**Pull request by LamboCreeper**
I've moved a lot of components to be styled by UIkit over our own custom styles. This allows us to build the site quicker (and with a better look in my opinion).

### CODE REVIEW

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 38
- Added 1
           

Complexity increasing per file
==============================
- src/components/organisms/CardGroup.js  1
- src/components/molecules/Card.js  1
         

Complexity decreasing per file
==============================
+ src/components/molecules/Navigation.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5655216&bid=18514464)

File: src/components/pages/Index.js [hunk 1]
```diff
@@ -4,14 +4,88 @@ import PageTemplate from "../templates/PageTemplate";
...
+        description: "CodeSupport's website backend API service which is powered by Spring Boot.",
+        repository: "https://github.com/codesupport/website-backend"
+    },
+    {
+        title: "Resources API",
+        language: "JSON",
+        description: "A simple repository containing various resources that the community recommends.",
+        repository: "https://github.com/codesupport/resources-api"
+    }
+]
```
Comment (LamboCreeper) [inline src/components/pages/Index.js:35]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5655216)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 38
           

Complexity increasing per file
==============================
- src/components/molecules/Card.js  1
- src/components/organisms/CardGroup.js  1
         

Complexity decreasing per file
==============================
+ src/components/molecules/Navigation.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5655216&bid=18514464)

**Review by Jason2605 — APPROVED** [review]
LGTM

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 58
           

Complexity increasing per file
==============================
- src/components/molecules/Card.js  1
- src/components/organisms/CardGroup.js  1
         

Complexity decreasing per file
==============================
+ src/components/molecules/Navigation.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5655216&bid=18514464)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 58
           

Complexity increasing per file
==============================
- src/components/molecules/Card.js  1
- src/components/organisms/CardGroup.js  1
         

Complexity decreasing per file
==============================
+ src/components/molecules/Navigation.js  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5655216&bid=18514464)

**Pull request merged**